.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
+}
```

Comment:
It might also be nice to add the `transform-class-properties` plugin to the .babelrc file

Reply:
Done! @veeqtor @anneKay